syntax = "proto2";

package romabuf;

message Peer {
}

message PleromaMessage {
  required int32 vat_id = 1;
  required int32 actor_id = 2;
  required int32 function_id = 3;
}

message AnnouncePeer {
  required int32 address = 1;
}

message Greeting {
  required string node_name = 1;
}

message GreetingAck {
  required int32 node_id = 1;
}

message LoadProgram {
}

